playbook.yml

```diff
@@ -69,9 +69,12 @@ content:
       start_path: docs
 
     - url: https://github.com/OpenZeppelin/cairo-contracts
-      branches:
-        - release-v*
-        - '!release-v0.7.0-rc.0' # broken antora.yml
+      branches: ~
```

Comment:
Is this pattern not matching any branch?

Reply:
exactly: https://docs.antora.org/antora/latest/playbook/content-tags/#tags-key